podium-gateway/src/main/java/nl/thehyve/podium/domain/RequestDetail.java

```diff
@@ -81,7 +82,7 @@
     )
     @Fetch(FetchMode.JOIN)
     @BatchSize(size = 1000)
-    private Set<RequestType> requestType;
+    private Set<RequestType> requestType = new HashSet<>();
```

Comment:
Maybe Collections.emptySet() is a cleaner solution?

Reply:
Except for is that is an immutable collection, then you cannot anything anymore.

Reply:
https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#EMPTY_SET